web/app.js

```diff
@@ -2046,6 +2046,69 @@ function handleMouseWheel(evt) {
 window.addEventListener('DOMMouseScroll', handleMouseWheel);
 window.addEventListener('mousewheel', handleMouseWheel);
 
+function touchSwipeDetector(evt) {
```

Comment:
To prevent a possible error is the user manages to use a touch gesture before the viewer has finished initializing, let's add a check at the top of this function to prevent this:
```javascript
if (!PDFViewerApplication.initialized) {
  return;
}
```